about.html

```diff
@@ -0,0 +1,115 @@
+        </div>
+        <ul>
+          <li><a href="#">English</a></li>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </div>
+        <img src="./images/menu.png" class="menu" alt="menu-icon">
+        <div class="desktop-items">
+          <a href="index.html"><img src="./images/logo.png" alt="" class="desktop-logo"></a>
+          <ul>
+            <li><a href="about.html" class="about">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Sponsors</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="campaign">AA Campaign</a></li>
+          </ul>
+        </div>
+      <nav class="hidden">
+        <img src="./images/cancel.png" alt="" class="cancel">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Feature</a></li>
+        </ul>
+      </nav>
+      <main>
+            
+        <h1 class=" about-title">Africa Agriculture Summit 2023</h1>
+
+      </main>
+    </header>
+    <main>
+        <section class="location">
+            <div class="about-intro-container">
+                <p class="about-intro">The AfricaAgriculture Summit is the premier sustainability event for agriculture and by agriculture, convening the collective food and agriculture value chain to learn, develop and advance a shared vision for a sustainable and resilient Africa food system.</p>
+                </div>
+                <p class="contact-text">Please contact for any further questions <br> about Africa Agriculture Summit 2023</p>
+                <p class="email">wasiufasasi@gmail.com</p>
+                <div class="email-line"></div>
+                <div class="location-container">
+                    <h3 class="location-h3">Location</h3>
+                    <div class="location-line"></div>
+                    <img src="./images/logo.png" alt="logo">
+                    <p class="location-date">23 Dec 2023</p>
+                    <p class="about-venue">Lagos Continental Hotel, Lag, Nig.</p>
+                    
+                </div>
+        </section>
+        <div class="separation-line"></div>
+        <section class="archives">
+            <h3>Archives</h3>
+            <div class="location-line"></div>
+            <p class="archives-check">Check previous Africa Agriculture Summit</p>
+            <div class="archives-img">
+                <div class="arch">
+                    <div class="arch-image">
+                        <img src="./images/archiveimg1.jpg" alt="2021-summit" class="archivesimg1"/>
+                    </div>
+                    <div class="arch-text">
+                      <h3>2021</h3>
+                        <p class="overlay">Africa Agriculture Summit 2021 in Uganda</p>
+                    </div>  
```

Comment:
- [ ] Please notice the bottom section images are organized in a vertical column only on the mobile display. On a desktop display, these images should be set side by side. Please rearrange your images to mimic the original design. 
### Your design 
![image](https://user-images.githubusercontent.com/69077061/227599714-52024d2b-1e9c-458b-a4fd-6bb5db698eb6.png)
### Original design 
![image](https://user-images.githubusercontent.com/69077061/227600195-cbd5ab6c-15b6-44b6-8a99-a6612b3028ee.png)